packages/customer-effort-score/src/customer-feedback-modal/index.tsx

```diff
@@ -27,7 +27,7 @@ import { __ } from '@wordpress/i18n';
  * @param {Function} props.recordScoreCallback Function to call when the results are sent.
  * @param {string} props.label                 Question to ask the customer.
  */
-function CustomerFeedbackModal( { recordScoreCallback, label } ) {
+function CustomerFeedbackModal({ recordScoreCallback, label }: { recordScoreCallback: (score: number, comments: string ) => void, label: string } ) {
 	const options = [
```

Comment:
If you're getting lint errors maybe check that your prettier plugin is autoformatting files with `ts` or `tsx` extension. I think @psealock  ran into this

Reply:
It was a VSC settings issue. I had the prettier plugin set for js files, so I added it as the default

Reply:
Thanks! Looks like I had the same issue with prettier ignoring these files.